Sources/xcodeproj/Objects/Project/PBXProject.swift

```diff
@@ -191,6 +191,44 @@ public final class PBXProject: PBXObject {
 
         return reference
     }
+    
+    
+    /// Adds a local swift package
+    ///
+    /// - Parameter path: Relative path to the swift package
+    /// - Parameter productName: The product to depend on
+    /// - Parameter target: Target to link package product to
+    /// - Parameter addFileReference: Include a file reference to the package (defaults to main group)
+    public func addLocalSwiftPackage(path: String,
+                                     productName: String,
+                                     target: PBXTarget? = nil,
+                                     addFileReference: Bool = true) -> XCSwiftPackageProductDependency {
+        let objects = try! self.objects()
```

Comment:
I'd add a `throws` to the method signature and bubble this up instead of resorting to `try!`.

Reply:
Should not be that the same for `addSwiftPackage` then?

Reply:
We should be consistent yeah, but thinking about it again, I think it makes sense to bubble them up.